specs/super_circuit.md

```diff
@@ -0,0 +1,39 @@
+# Super Circuit
+
+The zkEVM is composed of many circuits, each of which is responsible for checking some EVM aspect.
+These circuits need to be aggregated or combined together somehow. We could say that the Super
+Circuit is one of the possible strategies to combine all other sub-circuits.
+
+The Super Circuit approach is quite simple, all sub-circuits are put together and their tables are
+shared among all of them. It contains the following circuits:
+ - [x] EVM Circuit
+ - [x] State Circuit
+ - [ ] MPT Circuit
+ - [x] Keccak Circuit
+ - [x] Tx Circuit
+ - [x] Bytecode Circuit
+ - [x] Copy Circuit
+ - [ ] Block Circuit
+ - [x] PublicInputs Circuit
+
+ and the following tables:
+ - [x] RW Table
+ - [x] MPT Table
+ - [x] Keccak Table
+ - [x] Tx Table
+ - [x] Bytecode Table
+ - [x] Copy Table
+ - [x] Block Table
+
+The following diagram shows the relation between this circuits and tables:
+![](./super_circuit.png)
```

Comment:
- Missing lookup from Copy Circuit to RW Table
- I think the Copy Circuit doesn't do lookups to the Keccak Table
- Since you added the future Block Circuit, I think the diagram will be more accurate if the EVM Circuit does a lookup to the Block Table (instead of contains)